.golangci.yaml

```diff
@@ -42,7 +42,11 @@ linters:
     - unused
     - varcheck
     - whitespace
-
+run:
+  timeout: 4m
+  # skip-files:
```

Comment:
do we need this? it seems like dead configuration

Reply:
removed